src/middleware/logging.ts

```diff
@@ -11,7 +11,9 @@ export const logging = (
                 status: res.statusCode,
                 method: req.method,
                 path: req.path,
+                body: req.body,
```

Comment:
Just wondering if there's anything in `req.body` which we wouldn't want to be logging (this will be added for all requests), e.g. anything user identifiable. I don't think so currently, but might make it easy to log something unintentionally going forward. What do you think?

Reply:
That's a good point, I'll replace this with something more specific